cmd/upgrade/roles/cmd.go

```diff
@@ -475,7 +485,8 @@ func run(cmd *cobra.Command, argv []string) {
 		}
 	}
 
-	if isUpgradeNeedForAccountRolePolicies && mode == interactive.ModeAuto || isOperatorPolicyUpgradeNeeded {
+	if rolePolicyBindings != nil && isUpgradeNeedForAccountRolePolicies &&
+		mode == interactive.ModeAuto || isOperatorPolicyUpgradeNeeded {
 		newRolePolicyBindings, err := ocmClient.ListRolePolicyBindings(cluster.ID(), true)
 		if err != nil {
 			reporter.Warnf("Failed to get rolePolicyBindings after upgrade." +
```

Comment:
When testing locally, I get this output if I force error.
"Failed to get rolePolicyBindings after upgrade. Please ensure that the required policies are attached to the upgraded roles."
If the error is a specific error code, should we also follow this here?
https://github.com/openshift/rosa/pull/2152/files#diff-42e429a79b1a9b4a9fbf33a5dab5ad029f8365750589cd5a1e3402e4ff3ddea1R272